File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumSensorVolumePortionToDisplayValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;
import javax.annotation.Nonnull;

/**
 *  
 A writer that can write a value as a part of a sensor to display.
 

 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumSensorVolumePortionToDisplayValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    *  
    Writes the value as a part of a sensor to display.
    
    

    * @param value The portion of the sensor to display.
    */
    void writePortionToDisplay(@Nonnull CesiumSensorVolumePortionToDisplay value);
}

File: DotNet/CesiumLanguageWriter/Advanced/ICesiumSensorVolumePortionToDisplayValuePropertyWriter.cs
namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a part of a sensor to display.
    /// </summary>
    public interface ICesiumSensorVolumePortionToDisplayValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a part of a sensor to display.
        /// </summary>
        /// <param name="value">The portion of the sensor to display.</param>
        void WritePortionToDisplay(CesiumSensorVolumePortionToDisplay value);
    }
}